### ISSUE TITLE
Define API Version 3 and Create Tickets

### ISSUE THREAD

**Issue by nllong** [SEED-platform/seed#2081]
We are going to start working on version 2.2 of the SEED API. Need to determine which endpoints to move and which to rework. See this spreadsheet: https://docs.google.com/spreadsheets/d/1F0fDjvGccc8Tw_jiY3-BmJJ5IONRYPgpOoSooRavIfs/edit#gid=356371414

**Comment by adrian-lara** [comment]
@nllong In planning for some upcoming work, here's the high-level steps I envisioned for the whole process. Let me know your thoughts.

1. Build the v3 endpoints. (We started this already.)
2. Convert the FE to use v3, checking for updates made to v2/2.1 since a particular v3 was first built.
At this point, we would begin sunsetting versions < v3 - not supporting those api versions but keep them around.
From dev-perspective this is happening in pieces, since we're not converting all v2 to v3 in one PR, instead we'll be doing it, generally, by resource.
From user-perspective, this is happening for all endpoints within one release :fingers-crossed:.
3. After some defined time, begin deprecation of versions < v3.

We should probably also discuss what user communication looks like, but ideally, we should nail down our plan first.
